__tests__/components/editor/InputLiteral.test.js

```diff
@@ -16,7 +16,7 @@ const plProps = {
 }
 
 describe('<InputLiteral />', () => {
-  const wrapper = shallow(<InputLiteral {...plProps} />)
+  const wrapper = shallow(<InputLiteral {...plProps} id={10}/>)
```

Comment:
nice! i made this change in my PR as well, so i'll rebase if your PR gets merged first